include/libdnf/rpm/package.hpp

```diff
@@ -38,280 +38,435 @@ class Goal;
 
-    /// Reurn nevra withou epoch when epoch is 0
-    /// @replaces libdnf:libdnf/hy-package.h:function:dnf_package_get_nevra(DnfPackage * pkg)
+    /// @return RPM package NEVRA (Name-Epoch:Version-Release.Arch). If the Epoch is 0, it is omitted from the output.
+    /// @since 5.0
+    //
+    // @replaces libdnf:libdnf/hy-package.h:function:dnf_package_get_nevra(DnfPackage * pkg)
     std::string get_nevra() const;
 
-    /// Reurn always nevra with epoch
-    /// @replaces libdnf:libdnf/hy-package.h:function:dnf_package_get_nevra(DnfPackage * pkg)
+    /// @return RPM package NEVRA (Name-Epoch:Version-Release.Arch). The Epoch is always present even if it is 0.
+    /// @since 5.0
+    //
+    // @replaces libdnf:libdnf/hy-package.h:function:dnf_package_get_nevra(DnfPackage * pkg)
     std::string get_full_nevra() const;
 
-    /// @replaces dnf:dnf/package.py:attribute:Package.group
-    /// @replaces libdnf:libdnf/hy-package.h:function:dnf_package_get_group(DnfPackage * pkg)
+    /// @return RPM package Group (RPMTAG_GROUP)
+    /// @since 5.0
+    //
+    // @replaces dnf:dnf/package.py:attribute:Package.group
+    // @replaces libdnf:libdnf/hy-package.h:function:dnf_package_get_group(DnfPackage * pkg)
     std::string get_group() const;
 
-    /// If package is installed, return get_install_size(). Return get_download_size() otherwise.
-    /// @replaces dnf:dnf/package.py:attribute:Package.size
-    /// @replaces libdnf:libdnf/hy-package.h:function:dnf_package_get_size(DnfPackage * pkg)
+    /// @return If package is installed, return get_install_size(). Return get_package_size() otherwise.
+    /// @since 5.0
+    //
+    // TODO(dmach): drop and use get_install_size()/get_package_size() instead?
+    // @replaces dnf:dnf/package.py:attribute:Package.size
+    // @replaces libdnf:libdnf/hy-package.h:function:dnf_package_get_size(DnfPackage * pkg)
     unsigned long long get_size() const;
 
-    /// Return size of an RPM package: <size package="..."/>
-    /// @replaces dnf:dnf/package.py:attribute:Package.downloadsize
-    /// @replaces libdnf:libdnf/hy-package.h:function:dnf_package_get_downloadsize(DnfPackage * pkg)
-    unsigned long long get_download_size() const;
-
-    /// Return size of an RPM package installed on a system: <size installed="..."/>
-    /// @replaces dnf:dnf/package.py:attribute:Package.installsize
-    /// @replaces libdnf:libdnf/hy-package.h:function:dnf_package_get_installsize(DnfPackage * pkg)
+    /// @return File size of the RPM package
+    /// @since 5.0
+    //
+    // @replaces dnf:dnf/package.py:attribute:Package.downloadsize
+    // @replaces libdnf:libdnf/hy-package.h:function:dnf_package_get_downloadsize(DnfPackage * pkg)
+    unsigned long long get_package_size() const;
+
+    /// @return Size the RPM package occupies when installed on disk (RPMTAG_LONGSIZE)
+    /// @since 5.0
+    /// @note The actual size on disk may vary based on block size and filesystem overhead
```

Comment:
I've rephrased the docstring to:
```
    /// @return Size the RPM package should occupy after installing on disk (RPMTAG_LONGSIZE)
    /// @since 5.0
    /// @note The actual size on disk may vary based on block size and filesystem overhead.
    ///       Libdnf doesn't provide any method to compute the actual size on disk.
```

Reply:
So if the package isn't downloaded/installed, this information isn't available, am I right? Come think of it this applies to all information pulled from the rpm header. Should this also be mentioned in the docs?